app/src/main/res/layout/activity_on_boarding.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
why do we need a constraint layout? this can be easily done without constraint layout

Reply:
@aggarwalpulkit596 i didn't know i can't use constraint layout.

Do i need to change it to linear layout only ??